libraries/AP_HAL_ChibiOS/hwdef/JHEMCU-H743HD/hwdef.dat

```diff
@@ -0,0 +1,180 @@
+
+# hw definition file for processing by chibios_hwdef.py
+# for JHEH743_HD hardware.
+# thanks to betaflight for pin information
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1411
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+MCU_CLOCKRATE_MHZ 480
+
+FLASH_SIZE_KB 1024
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 384
+
+define HAL_STORAGE_SIZE 16384
+define STORAGE_FLASH_PAGE 1
+
+STM32_ST_USE_TIMER 2
+
+# SPI devices
+
+# SPI1
+PA5 SPI1_SCK SPI1
+PA6 SPI1_MISO SPI1
+PA7 SPI1_MOSI SPI1
+
+# SPI2
+PB13 SPI2_SCK SPI2
+PB14 SPI2_MISO SPI2
+PB15 SPI2_MOSI SPI2
+
+# SPI3
+PB2 SPI3_MOSI SPI3
+PC10 SPI3_SCK SPI3
+PC11 SPI3_MISO SPI3
+
+# SPI4
+PE12 SPI4_SCK SPI4
+PE13 SPI4_MISO SPI4
+PE14 SPI4_MOSI SPI4
+
+# Chip select pins
+PA15 FLASH1_CS CS
+PB12 OSD1_CS CS
+PA4 GYRO1_CS CS
+PE11 GYRO2_CS CS
+
+# Beeper
+PE3 BUZZER OUTPUT GPIO(80) LOW
+define HAL_BUZZER_PIN 80
+
+# SERIAL ports
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 UART5 USART6 UART7 UART8 OTG2
+# PA10 IO-debug-console
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# USART1
+PA10 USART1_RX USART1
+PA9 USART1_TX USART1
+
+# USART2
+PD5 USART2_TX USART2
+PD6 USART2_RX USART2
+define DEFAULT_SERIAL2_PROTOCOL SerialProtocol_RCIN
+
+# USART3
+PD8 USART3_TX USART3
+PD9 USART3_RX USART3
+define DEFAULT_SERIAL3_PROTOCOL None
+
+# UART4
+PD0 UART4_RX UART4
+PD1 UART4_TX UART4
+define DEFAULT_SERIAL4_PROTOCOL SerialProtocol_GPS
+
+# UART5
+PC12 UART5_TX UART5 NODMA
+PD2 UART5_RX UART5 NODMA
+
+# USART6
+PC6 USART6_TX USART6
+PC7 USART6_RX USART6
+define DEFAULT_SERIAL6_PROTOCOL SerialProtocol_DJI_FPV
```

Comment:
HD is more often DisplayPort these days, DJI WTFOS, Walksnail, etc. I am suggesting using it instead of DJI_FPV default
```suggestion
define DEFAULT_SERIAL6_PROTOCOL SerialProtocol_MSP_DisplayPort
```